Supports 1.13.2

ncurran02 opened the pull request:
This also resolves #13

ncurran02 committed:
Starting to make Aero compatiable with 1.13

It has not been tested so [BLEEDING]

ncurran02 committed:
1.13.2

ncurran02 committed:
idk

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/Pravian/Aero?pullRequest=14) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/Pravian/Aero?pullRequest=14) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/Pravian/Aero?pullRequest=14) it.</sub>

ncurran02 committed:
Formatting

ncurran02 committed:
Spacing

JeromSar commented on src/main/java/net/pravian/aero/util/Materials.java:
```diff
@@ -11,56 +11,56 @@
-        MATERIALS.add(Material.LAVA.getId());
+        final HashSet<String> MATERIALS = new HashSet<>();
+        MATERIALS.add(Material.AIR.toString());
+        MATERIALS.add(Material.LEGACY_SAPLING.toString());
```
Considering Bukkit's `Material` class has changed a bit. Would you please switch to the new items values (i.e. avoid legacy)?

Focusvity replied:
Would you prefer if I have 2 sets? One for new items value and one for legacy?

JeromSar replied:
I don't think we'll need the legacy items. If I'm not mistaken, the legacy items are just here for backward compatibility for some functionality.

JeromSar reviewed (changes requested):
There are truly only minor edits here and there. Most of it seems in good order.

Could you please:
1. Use Google's formatting guidelines? A IntelliJ plugin which does this automatically is: https://plugins.jetbrains.com/plugin/8527-google-java-format
2. Avoid double paragraph tags (`<p>`) in the javadocs?
3. Ensure the class member order is consistent (e.g. static methods after instance methods).

ncurran02 committed:
Code clean-up

Updated Materials class - removed legacy and added 1.13 materials
Formatting
Fix Javadocs

Focusvity commented:
@JeromSar, is there any other changes needed?

Focusvity commented:
Hey @JeromSar,
Is there anything you need me to change?

JeromSar commented:
We're up to date to 1.15.2 now. :)
